maven-javadoc-plugin/src/main/java/org/apache/maven/plugin/javadoc/AbstractJavadocMojo.java

```diff
@@ -288,6 +289,8 @@
      */
     private static final float SINCE_JAVADOC_1_8 = 1.8f;
 
+    private static final float JAVA_VERSION_FLOAT = JavadocUtil.parseJavadocVersion(SystemUtils.JAVA_VERSION);
```

Comment:
@michael-o I just noticed there is a new [`Runtime.Version`](http://download.java.net/java/jdk9/docs/api/java/lang/Runtime.Version.html) class in Java 9 which can parse and compare Java version strings. Do you think we can use with reflection in place of `JavaVersion`?

But as far as upgrading to commons-lang3, having a new method `float parseJavaVersion(String)`, that is similar to `parseJavadocVersion` but with the regex being `([0-9]+\\.?[0-9]*)(\\.([0-9]+))`, looks like the way to go. We can look at using `Runtime.Version` or not later.